src/routes/projects/[project]/strategise/_components/BudgetInput.svelte

```diff
@@ -33,8 +34,11 @@
 				{/snippet}
 			</Form.Control>
 			<Form.Description class="text-xs"
-				>Enter your maximum possible budget between <span class="font-semibold">${minCost.toLocaleString()}</span> and
-				<span class="font-semibold">${maxCost.toLocaleString()}</span>.
+				>Enter your maximum possible budget between <span class="font-semibold"
+					>${convertToLocaleString(minCost, 0, 'ceil')}</span
+				>
+				and
+				<span class="font-semibold">${convertToLocaleString(maxCost, 0, 'ceil')}</span>.
```

Comment:
displaying ceil number for both... as its okay if they put a number above the max budget